ts/Core/Defaults.ts

```diff
@@ -256,6 +276,30 @@ const defaultOptions: DefaultOptions = {
          */
         numericSymbols: ['k', 'M', 'G', 'T', 'P', 'E'],
 
+        /**
+         * The default name for a pie slice (point).
+         * @since next
+         */
+
+        pieSliceName: 'Slice',
+
+        /**
+         * The default text for the rangeselector buttons.
+         *
+         * @since next
+         */
+
+        rangeSelector: {
+            allText: 'All',
+            allTitle: 'View all',
+            monthText: '{count}m',
+            monthTitle: 'View {count} {#eq count 1}month{else}months{/eq}',
+            yearText: '{count}y',
+            yearTitle: 'View {count} {#eq count 1}year{else}years{/eq}',
+            ytdText: 'YTD',
+            ytdTitle: 'View year to date'
+        },
```

Comment:
Also specific to Stock

Reply:
See [RangeSelectorDefaults](https://github.com/highcharts/highcharts/blob/4e64984bbde6fe6c825708fe57d6d98f1e8fb59e/ts/Stock/RangeSelector/RangeSelectorDefaults.ts#L44) for an example of extending `lang` in a module.